tasks/jasmine.js

```diff
@@ -194,7 +195,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.showLogging) {
```

Comment:
Simply defaulting `showLogging` to `undefined` does not provide backwards compatibility. I alluded in an [earlier comment](https://github.com/gruntjs/grunt-contrib-jasmine/pull/236#discussion_r67083405) that `showLogging = undefined` could be explicitly handled to provide backwards compatibility.

```js
var fullLogging = (typeof(options.showLogging) == 'undefined' && options.display == 'full');
if (options.showLogging || fullLogging) {
```